queries.sql

```diff
@@ -38,7 +38,6 @@ ROLLBACK TO dateOfBirth;
 UPDATE animals SET weight_kg=-1*weight_kg WHERE weight_kg<0;
 COMMIT; 
 
-
 SELECT COUNT(*) FROM animals;
 SELECT COUNT(*) FROM animals WHERE escape_attempts=0;
 SELECT AVG(weight_kg) FROM animals;
```

Comment:
- [ ] Optional suggestions
Instead of writing this way,

DELETE FROM animals WHERE date_of_birth>'2022-01-01';
SAVEPOINT dateOfBirth;
UPDATE animals SET weight_kg=-1*weight_kg;
ROLLBACK TO dateOfBirth;
UPDATE animals SET weight_kg=-1*weight_kg WHERE weight_kg<0;

I would recommend this one to be much more clean and clear:

DELETE FROM animals WHERE date_of_birth>'2022-01-01';
SAVEPOINT dateOfBirth;
UPDATE animals SET weight_kg= weight_kg * -1;
ROLLBACK TO dateOfBirth;
UPDATE animals SET weight_kg= weight_kg * -1 WHERE weight_kg<0;

Reply:
I will work on it right now